interfaces/cython/cantera/cti2yaml.py

```diff
@@ -0,0 +1,1648 @@
+    a = atoms.replace(',',' ')
+    toks = a.split()
+    d = OrderedDict()
+    for t in toks:
+        b = t.split(':')
+        try:
+            d[b[0]] = int(b[1])
+        except ValueError:
+            d[b[0]] = float(b[1])
+    return d
+
+
+class element(object):
+    """ An atomic element or isotope. """
+    def __init__(self, symbol='', atomic_mass=0.01, atomic_number=None):
+        """
+        :param symbol:
+            The symbol for the element or isotope.
+        :param atomic_mass:
+            The atomic mass in amu.
+        """
+        self._sym = symbol
+        self._atw = atomic_mass
+        self._num = atomic_number
+        _elements.append(self)
+
+    @classmethod
+    def to_yaml(cls, representer, node):
+        out = BlockMap([('symbol', node._sym),
+                        ('atomic-weight', node._atw)])
+        if node._num is not None:
+            out['atomic-number'] = node._num
+        return representer.represent_dict(out)
+
+
+class species(object):
+    """A constituent of a phase or interface."""
+
+    def __init__(self, name, atoms='', note = '', thermo=None, transport=None,
+                 charge=None, size=1.0, standardState=None):
+        """
+        :param name:
+            The species name (or formula). The name may be arbitrarily long,
+            although usually a relatively short, abbreviated name is most
+            convenient. Required parameter.
+        :param atoms:
+            The atomic composition, specified by a string containing
+            space-delimited ``<element>:<atoms>`` pairs. The number of atoms may be
+            either an integer or a floating-point number.
+        :param note:
+            A user-defined comment. Not evaluated by Cantera itself.
+        :param thermo:
+            The parameterization to use to compute the reference-state
+            thermodynamic properties. This must be one of the entry types
+            described in `Thermodynamic Property Models
+            <https://cantera.org/science/science-species.html#sec-thermo-models>`_.
+            To specify multiple parameterizations, each for a different temperature range,
+            group them in parentheses.
+        :param transport:
+            An entry specifying parameters to compute this species'
+            contribution to the transport properties. This must be one of the
+            entry types described in `Species Transport Coefficients
+            <https://cantera.org/science/science-species.html#species-transport-coefficients>`_,
+            and must be consistent with the transport model of the phase into which
+            the species is imported. To specify parameters for multiple
+            transport models, group the entries in parentheses.
+        :param size:
+            The species "size". Currently used only for surface species,
+            where it represents the number of sites occupied.
+        :param charge:
```

Comment:
If you want to fix the camelCasing of the new classes / arguments associate with the addition of the `BinarySolutionTabulatedThermo` class, it will have to be done both here and in `ctml_writer`. I think we *can* make this change, since the class is new with this version of Cantera, but this should probably be brought up with @decaluwe.

Reply:
OK, I'll opened #658